### ISSUE TITLE
Netbeans 11.3

### ISSUE THREAD

**Issue by RolandSM** [vralfy/phpcsmd#49]
Hello,

I used since years this great extension to see all action items when a file was loaded / saved.

But in currently 11.3 it stops working?

Really or do I have configuration errors?

**Comment by ccerrillo** [comment]
@RolandSM did you managed it?

**Comment by RolandSM** [comment]
Yes. It was a failure of my local installed Codesniffer.
The Plugin works...

**Comment by RolandSM** [comment]
Have to open it again, try to get it with Netbeans 12.1 to work but no action items with working code sniffer???

**Comment by RolandSM** [comment]
Hello,

I have tried more things why this is not working. I updated the phpcs to latest 3.5.8, same error.
I activated the debug log, set to info and show.

The command is: (example)
`/usr/bin/phpcs --standard=MYSTANDARD -w --report=xml /home/MYUSER/GIT\ Development/MYPROJECT/sub/folder/myfile.php `

The log shows also an error but I think the real error is another:
ERROR: The file "/home/MYUSER/GIT\" does not exist. Seems the space is a trouble?

But I have taken the command and put it into the Ubuntu console and there also the file is not available...

I wrote sign by sign the line again in command line and then it worked?

Very confused...

Then I put both into an text editor and saw, that the command from phpcsmd debug log have no spaces but other signs???

See in image
![netbeans](https://user-images.githubusercontent.com/16316603/100133106-a42f7100-2e86-11eb-855b-1a317dea93c5.png)
